src/gt4py/next/program_processors/runners/dace/workflow/bindings.py

```diff
@@ -84,48 +65,24 @@ def _parse_gt_param(
     For tuple arguments, this function is recursively called on all elements of the tuple.
     """
     if isinstance(param_type, ts.TupleType):
-        # Special handling of tuples
-        if (m := FIELD_RANGE_PARAM_RE.match(param_name)) is not None:
-            # Domain range is expressed as a tuple in each dimension
-            gt_field_name, dim_value = m[1], m[2]
-            dim = gtx_common.Dimension(dim_value)
-            rstart = gtx_dace_utils.range_start_symbol(gt_field_name, dim)
-            rstop = gtx_dace_utils.range_stop_symbol(gt_field_name, dim)
-            for i, tuple_param_name in enumerate([rstart, rstop]):
-                tuple_arg = f"{arg}[{i}]"
-                tuple_param_type = param_type.types[i]
-                assert isinstance(tuple_param_type, ts.ScalarType)
-                _parse_gt_param(
-                    tuple_param_name,
-                    tuple_param_type,
-                    tuple_arg,
-                    code,
-                    sdfg_arglist,
-                    make_persistent,
-                )
-        else:
-            # For regular data tuples, each element of the tuple gets a name
-            # with an index-based suffix and it is recursively visited.
-            for i, tuple_param_type in enumerate(param_type.types):
-                tuple_arg = f"{arg}[{i}]"
-                tuple_param_name = f"{param_name}_{i}"
-                assert isinstance(tuple_param_type, ts.DataType)
-                _parse_gt_param(
-                    tuple_param_name,
-                    tuple_param_type,
-                    tuple_arg,
-                    code,
-                    sdfg_arglist,
-                    make_persistent,
-                )
+        # Each element of a tuple gets a name with an index-based suffix and it is recursively visited.
+        tuple_args = [f"{arg}_{i}" for i in range(len(param_type.types))]
+        code.append("{} = {}".format(", ".join(tuple_args), arg))
```

Comment:
```suggestion
        code.append(f"{', '.join(tuple_args)} = {arg}"
```
or 
```suggestion
        code.append(f"{str.join(', ', tuple_args)} = {arg}"
```
Whatever your prefer, both are equivalent. The first option is more common but I lately started using the second one, since I find it more readable.